controllers/customers.js

```diff
@@ -0,0 +1,48 @@
+const models = require('../models')
+const Customer = require('../models/customer')
+
+
+const handleUploadCustomers = customers => {
+  const customerEmailValidate = customers.map(customer => {
+    const { email } = customer
+    // removes whitespaces and commas
+    const invalidChars = /[,\s]+|[,\s]+/g
+
+    const validatedEmail = email.replace(invalidChars, '')
+
+    return {
+      ...customer,
+      'email': validatedEmail,
+    }
+  })
+
+  Customer.BulkCreate(customerEmailValidate)
+}
+
+const getAllCustomers = async (req, res) => {
+  const customers = await models.Customer.findAll()
+
+  return res.send(customers)
+}
+
+const getCustomerById = async (req, res) => {
+  const { Id } = req.params
```

Comment:
Naming conventions in JS, says we should use camelcase variables. These camel case variables start with lowecase